homeassistant/components/homematicip_cloud/sensor.py

```diff
@@ -65,6 +65,20 @@ def __init__(self, home):
         """Initialize access point device."""
         super().__init__(home, home)
 
+    @property
+    def device_info(self):
+        """Return device specific attributes."""
+        return {
+            'identifiers': {
+                # Serial numbers of Homematic IP device
+                (HMIPC_DOMAIN, self._device.id)
+            },
+            'name': self._device.label,
+            'manufacturer': 'eQ-3',
```

Comment:
Why not use `device.oem` ?

Reply:
The hub has no oem. In homematicip it is called access point. It's the only device, that is not a child of homematicip's device class, but it is physical.

Reply:
If this is the hub, why reference it as `device.id`, and above you use `device.homeId`, are those the same?

Reply:
It's the same. But now hub/ap device creation moved to __init__.py